addons/hr_recruitment/migrations/13.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,27 @@
+---Models in module 'hr_recruitment'---
+---Fields in module 'hr_recruitment'---
+hr_recruitment / hr.applicant             / email_cc (text)               : type is now 'char' ('text')
```

Comment:
in pre-migration:

```python
openupgrade.logged_query(
        env.cr, """
        ALTER TABLE hr_applicant
        ALTER COLUMN email_cc
        TYPE varchar"""
    )
```

although maybe not needed, I have not tested if it is done automatically without losing data

Reply:
I added the code in case if didn't do it automatic